shared/AppInsightsCore/src/JavaScriptSDK/CookieMgr.ts

```diff
@@ -279,6 +323,12 @@ export function createCookieMgr(rootConfig?: IConfiguration, logger?: IDiagnosti
             let value = STR_EMPTY
             if (_isMgrEnabled(cookieMgr) && !_isIgnoredCookie(cookieMgrConfig, name)) {
                 value = _getCookieFn(name);
+            } else if (!_isIgnoredCookie(cookieMgrConfig, name) && _pendingCookies[name] && _pendingCookies[name].operation === 'set') {
```

Comment:
Remove additional call for the _isIgnoreCookie, only call it once in this function